### PR TITLE
[INS-1954] Fix message being removed when switching environments

### PR DESCRIPTION
**Pull request by marckong**
- Removing a React state that controls the CodeMirror.setValue execution

https://www.loom.com/share/ba9254568c2440a2b528fbac137b6aa2

Closes [INS-1954](https://linear.app/insomnia/issue/INS-1954/[bug]-message-edits-disappear-after-switching-environments-in), INS-1952

### CODE REVIEW

**Review by filfreire — APPROVED** [review]
Tested locally, LGTM!

Now edits to message don't disappear when switching between environments.

**Review by gatzjames — APPROVED** [review]
Nice one!
That was also the root cause for the JSON validation running on empty state!

File: packages/insomnia/src/ui/components/websockets/websocket-request-pane.tsx [hunk 1]
```diff
@@ -70,20 +68,19 @@ const PayloadTabPanel = styled(TabPanel)({
...
   const editorRef = useRef<UnconnectedCodeEditor>(null);
+
   useEffect(() => {
-    let isMounted = true;
-    if (isMounted) {
-      editorRef.current?.codeMirror?.setValue(initialValue);
+    async function initMessageText(): Promise<void> {
+      const payload = await models.webSocketPayload.getByParentId(request._id);
+      const msg = payload?.value || '';
+      editorRef.current?.codeMirror?.setValue(msg);
```
Comment (jackkav) [inline packages/insomnia/src/ui/components/websockets/websocket-request-pane.tsx:79]: we probably want to add a isMounted check here at some point

File: packages/insomnia/src/ui/components/websockets/websocket-request-pane.tsx [hunk 2]
```diff
@@ -117,6 +114,19 @@ const WebSocketRequestForm: FC<FormProps> = ({
...
+    if (payload) {
+      await models.webSocketPayload.update(payload, { value });
+    } else {
+      await models.webSocketPayload.create({
+        parentId: request._id,
+        value,
+        mode: previewMode,
+      });
+    }
+  };
```
Comment (jackkav) [inline packages/insomnia/src/ui/components/websockets/websocket-request-pane.tsx]: would be nice to make it clear in the name how this createOrUpdate is different from the nearly identical one 50 lines below.

**Review by jackkav — APPROVED** [review]

File: packages/insomnia/src/ui/components/websockets/websocket-request-pane.tsx (hunk 1 above)
Comment (marckong) [reply packages/insomnia/src/ui/components/websockets/websocket-request-pane.tsx:79]: I think here the root cause was the isMounted. I assume the isMounted check was needed for some re-rendering related issue. I would propose that we should fix that issue. It feels isMounted check may be a workaround the root cause.

**Pull request merged**